.sync.yml

```diff
@@ -21,34 +22,34 @@ appveyor.yml:
   unmanaged: true
 Gemfile:
   required:
-    ':development':
-      - gem: 'puppet-lint-absolute_classname-check'
+    ":development":
+      - gem: "puppet-lint-absolute_classname-check"
```

Comment:
Is there a reason you changed all of these? i'm sure its not a problem but typically if you are doing interpolation you would use single quotes.